docs/node-port-local.md

```diff
@@ -137,6 +138,7 @@ annotation is deprecated. The array contains a single member, equal to the
 `protocol` field.
 The `protocols` field will be removed from Antrea for minor releases post March 2023,
```

Comment:
Hi @jianjuns @antoninbas @tnqn  according to the doc, this field is deprecated almost a year, could you suggest if we should remove the field `protocols` in Antrea v2.0? Thanks.

Reply:
Yes I think we can remove it.